tests/objects/expression.js

```diff
@@ -1,3 +1,7 @@
 () => ({}``);
 ({})``;
 a = () => ({}).x;
+({} && a, b);
+({}::b, 0);
+({}::b()``[''].c++ && 0 ? 0 : 0, 0);
+({}(), 0);
```

Comment:
Could you add `(({} = 0);` and `(({} = 0), 1);`? They pass because of #862, but it'd be good to ensure they continue to.